samples/nrf9160/cloud_simple/CMakeLists.txt

```
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+cmake_minimum_required(VERSION 3.8.2)
+
+include(../../../cmake/boilerplate.cmake NO_POLICY_SCOPE)
```

This should be changed to follow [PR#1680](https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/1680)
```suggestion
include($ENV{ZEPHYR_BASE}/../nrf/cmake/boilerplate.cmake NO_POLICY_SCOPE)
```